.gitignore

```diff
@@ -1,20 +1 @@
-*.py[cod]
-
-# Packages
-*.egg
-*.egg-info
-dist
-build
-eggs
-parts
-var
-sdist
-develop-eggs
-.installed.cfg
-lib
-lib64
-
-# python virtual environments for testing
-venv/
-
-_build/
+.DS_Store
```

Comment:
this should be part of a global gitignore: https://github.com/deis/deis/pull/2566